<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PrintInitDialog</class>
 <widget class="QDialog" name="PrintInitDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>759</width>
    <height>511</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>3D Print</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QTextBrowser" name="textBrowser">
     <property name="minimumSize">
      <size>
       <width>480</width>
       <height>128</height>
      </size>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="source">
      <url>
       <string>qrc:/html/PrintInitDialog.html</string>
      </url>
     </property>
     <property name="openExternalLinks">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string/>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="topMargin">
         <number>0</number>
        </property>
        <item>
         <layout class="QHBoxLayout" name="printServiceLayout">
          <item>
           <widget class="QPushButton" name="pushButtonLocalApplication">
            <property name="focusPolicy">
             <enum>Qt::TabFocus</enum>
            </property>
            <property name="text">
             <string>Run local Application</string>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
            <property name="autoDefault">
             <bool>false</bool>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">buttonGroup</string>
            </attribute>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButtonOctoPrint">
            <property name="focusPolicy">
             <enum>Qt::TabFocus</enum>
            </property>
            <property name="text">
             <string>OctoPrint</string>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
            <property name="autoDefault">
             <bool>false</bool>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">buttonGroup</string>
            </attribute>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="horizontalSpacerPrintService">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayoutFileFormat">
        <item>
         <widget class="QLabel" name="labelFileFormat">
          <property name="text">
           <string>File format</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBoxFileFormat"/>
        </item>
        <item>
         <spacer name="horizontalSpacerFileFormat">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBoxEnableRemotePrintServices">
        <property name="text">
         <string>Enable remote services that need network access</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayoutDialogButtons">
     <property name="topMargin">
      <number>9</number>
     </property>
     <item>
      <widget class="QCheckBox" name="checkBoxAlwaysShowDialog">
       <property name="text">
        <string>Always show dialog</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacerButtons">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonCancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonOk">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>pushButtonOk</tabstop>
  <tabstop>pushButtonLocalApplication</tabstop>
  <tabstop>pushButtonOctoPrint</tabstop>
  <tabstop>comboBoxFileFormat</tabstop>
  <tabstop>checkBoxEnableRemotePrintServices</tabstop>
  <tabstop>checkBoxAlwaysShowDialog</tabstop>
  <tabstop>pushButtonCancel</tabstop>
 </tabstops>
 <resources>
  <include location="../../resources/common.qrc"/>
 </resources>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
